assets/data/cards_pt-br.json

```
@@ -482,6 +482,72 @@
         "Conceito",
         "Back-end"
       ]
+    },
+    {
+      "title": "NaN",
+      "description": "A propriedade global NaN é um valor especial que significa Not-A-Number(Não é um número).",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Null",
+      "description": "Null é um literal que representa um valor nulo ou vazio.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Undefined",
+      "description": "O valor global undefined representa um valor indefinido, ou seja, um valor que não foi definido ou atribuído em uma variável.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Falsy",
+      "description": "Falsy é um valor que se traduz por falso quando avaliado em um contexto Booleano.",
+      "content": {
+        
+        "code": "São considerados valores Falsy(false, null, undefined, 0, [], '')"
+      },
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Truthy",
+      "description": "Truthy é um valor que se traduz por falso quando avaliado em um contexto Booleano.",
```

"truthy" é o oposto de "falsy", veja se essa explicação faz sentido pra você:

```suggestion
      "description": "Truthy é um valor que é considerado como verdadeiro quando avaliado em uma situação que espera um valor verdadeiro ou falso.",
```